### DISCUSSION TITLE
Edit Telegram Bot

### DISCUSSION THREAD

**Discussion by Erfantls** [FranzKafkaYu/x-ui#224] [General]
How can I edit the Telegram bot? what file?
For example, edit the word "功能菜单" on telegram bot to English

**Comment by Vahid-Spacer** [comment]
Please tell me too؟!؟!؟!

**Comment by lahirubro123** [comment]
i need tooo

**Comment by FranzKafkaYu** [comment]
sorry I didn't release the code,if you have any questions, please contact me: loveme@duck.com

**Reply by mehrdadking70** [reply]
Please check your email, I sent you an email

**Reply by Erfantls** [reply]
chizi peyda kardi ? ya soal kardi ? jvb email dir be dir mide

**Comment by Erfantls** [comment]
thank you for your replay
